taca/utils/undetermined.py

```diff
@@ -0,0 +1,164 @@
+import re
+import subprocess
+import gzip
+import glob
+import os
+import logging
+import taca.flowcell_parser.classes as cl
```

Comment:
careful with this shortname, in `analysis.py` and others, cl is used as a variable name for command line, not saying that it **is** a problem, but surely it _can_ be